src/main/java/executor/service/config/bean/ThreadPoolExecutor.java

```diff
@@ -0,0 +1,56 @@
+package executor.service.config.bean;
+
+import executor.service.config.properties.PropertiesConfig;
+import executor.service.model.ThreadPoolConfig;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.TimeUnit;
+
+import static executor.service.config.properties.PropertiesConstants.*;
```

Comment:
This is a very good practice, it allows you to hide details that are not important.

Reply:
> not good practice to set star in imports

Explicit imports make your code more readable and clear. When someone reads your code, they can immediately see which classes are being used from which packages. This transparency helps in understanding the dependencies of your code. 
When using wildcard imports, there's a risk of naming conflicts if two or more classes with the same name exist in different packages. Explicit imports help avoid such conflicts by specifying the exact class you want to use.
Wildcard imports can lead to the inclusion of unnecessary classes, which may never be used in your code. This can bloat your compiled code and potentially slow down the compilation process.
If you decide to refactor your code, wildcard imports can lead to unintentional changes in behavior. For example, if you add a class to a package with a wildcard import, your code might suddenly use that new class, which could cause unexpected behavior.
During code reviews, explicit imports make it easier for other developers to understand and review your code. They can quickly identify which packages and classes are being used.